src/app/shell/About.tsx

```diff
@@ -88,6 +89,15 @@ export default function About() {
             </ExternalLink>
           </li>
         )}
+        <li
+          dangerouslySetInnerHTML={{
```

Comment:
huh???

Reply:
Yeah, we don't use react-i18next or anything, so if you want to inject HTML into a translated string, you get an HTML string back and thus you gotta use `dangerouslySetInnerHTML`. It's OK in this instance because we're constructing a static string, nothing out of our control.

Reply:
kinda wild but i guess i can't construct something using bullet points and i18n segments without it getting considerably more awkwardly phrased.